Sheet 2/ProblemF.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
+{
+    int a, b, d = 0;
+
+    cin >> a >> b;
+
+    for (int i = 0; i <= 32; i++)
+    {
+        int x = a % 2;
+        a = a / 2;
+
+        int y = b % 2;
+        b = b / 2;
+
+        if (x != y)
+        {
+            d = d + pow(2, i);
```

Comment:
When we're in the integer world, don't use `pow`, write yourself a power function that works with integers & powers by looping.